doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -361,3 +362,29 @@
     constexpr uint32_t MAX_VAL = algorithm::maxVal(3, 1890, 57);
     constexpr uint32_t MIN_VAL = algorithm::minVal(3, 1890, 57);
     ```
+
+20. The `CMakeLists.txt` of apps using iceoryx need to add `iceoryx_platform`
```

Comment:
It should be mentioned that the colcon build is broken with these  changes. A possible mitigation is the following:

```
cd src/iceoryx/iceoryx_hoofs/platform
cmake -Bbuild -DCMAKE_INSTALL_PREFIX=build/install -DBUILD_SHARED_LIBS=ON .
cmake --build build/ --target install
cd workspace_folder
colcon build --cmake-args -Diceoryx_platform_DIR=/path/to/iceoryx_workspace/src/iceoryx/iceoryx_hoofs/platform/build/install/lib/cmake/iceoryx_platform
```